hs_discover/src/App.vue

```diff
@@ -1,10 +0,0 @@
-<template>
```

Comment:
this was stub code for the initial dark code showing vue to the devs in older commits